### PR TITLE
Add OpenAI Embedding Model 'text-embedding-3-small'

### PR DESCRIPTION
**Pull request by bjorger**
As OpenAI has introduced a new embedding model, I added it in the OpenAITokenizer, so I can be used.

### CODE REVIEW

File: griptape/tokenizers/openai_tokenizer.py
Comment (collindutter) [inline griptape/tokenizers/openai_tokenizer.py:1]: Please add the model to this list as well.

Comment (collindutter) [inline griptape/tokenizers/openai_tokenizer.py:1]: Thanks for the PR! Would you mind also adding `large` and updating the `MODEL_PREFIXES_TO_MAX_TOKENS`?
image

**Review by collindutter — CHANGES_REQUESTED** [review]

File: griptape/tokenizers/openai_tokenizer.py
Comment (bjorger) [reply griptape/tokenizers/openai_tokenizer.py:1]: hey collin, sure thing!

Comment (bjorger) [reply griptape/tokenizers/openai_tokenizer.py:1]: Sure, will add :)

**Pull request closed without merging**